src/Templates/src/templates/maui-mobile/AppShell.xaml

```diff
@@ -7,7 +7,7 @@
     Shell.FlyoutBehavior="Disabled">
 
     <ShellContent
-        Title="Hello, World!"
+        Title=".NET MAUI"
```

Comment:
Maybe ".NET MAUI App"

Or ideally, is there a way we could directly reference the App Title here?